### ISSUE TITLE
[Question] When can we expect the image upload code go live with the present package?

### ISSUE THREAD

**Issue by Chandu-bs** [AppFlowy-IO/appflowy-editor#276]
### Description

Hey, Hi

Was waiting for the local image upload code to be featured in the new update. When can we expect this to happen? 
Thanks!

### Impact

Local Image Uploading featue

**Comment by LucasXu0** [comment]
Hey, @Chandu-bs. We have not yet introduced this feature to mobile; it is currently only supported on the desktop platform.

**Comment by Chandu-bs** [comment]
How can we access the web image, where I was getting an error.
```
Error:       On web `path` is always `null`,
      You should access `bytes` property instead,
```

Can we change implementation for web?

**Comment by LucasXu0** [comment]
> How can we access the web image, where I was getting an error.
> 
> ```
> Error:       On web `path` is always `null`,
>       You should access `bytes` property instead,
> ```
> 
> Can we change implementation for web?

Was this error raised on the web platform?

**Comment by Chandu-bs** [comment]
Yes.. I think we should convert path to bytes for web and change widget image.memory.. Will try this and update

**Comment by LucasXu0** [comment]
If it's for the web platform, another option is that we can use base64 to save the image.

**Comment by Chandu-bs** [comment]
Would like to know more about this. So converting the selected image file from web and converting it to base64 image for both displaying and saving?

**Comment by Chandu-bs** [comment]
Yes. Wanted to say that only, storing base64 image could increase the document size and more the images in editor, more the size

**Comment by LucasXu0** [comment]
I think it's related to this issue.https://github.com/flutter/flutter/issues/129161
